index.bs

```diff
@@ -697,16 +697,16 @@ The getter of the {{Sensor/state!!attribute}} attribute returns <emu-val>this</e
 The getter of the {{Sensor/timestamp!!attribute}} attribute returns
 [=latest reading=]["timestamp"].
 
-### Sensor.start() ### {#sensor-start}
+### Sensor.activate() ### {#sensor-activate}
```

Comment:
Would be nice to keep the previous id around (by using the [oldids](https://tabatkins.github.io/bikeshed/#id-gen) attribute. You'll have to use regular HTML for the title though, not markdown.